src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -29,7 +29,7 @@ public enum Formatting
         Indented = 1,
     }
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-    [System.ObsoleteAttribute("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", true)]
+    [System.ObsoleteAttribute("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.")]
```

Comment:
Is it OK to remove this `error = true`?

Reply:
cc @dotnet/area-system-xml

Reply:
We actually need to so that we can typeforward to this type from the System.Xml shim. The compiler does not allow a typeof(Type) statement when the type is marked as obsolete and the IsError parameter is true. That's why the source was different than the ref. The System.Xml shim used the source to generate the shim but now that we work towards unifying source and ref we can't allow such differences.

Reply:
I don't think that we should turn Error=true into Error=false.

That suggests one of:
* We need a different way of building the typeforwards
* To just acknowledge that not everything can be unified
* Something we can put in the source that makes GenAPI and/or the roslyn version change Obsolete(false) in src to Obsolete(true) in ref.

Reply:
See https://github.com/dotnet/corefx/pull/36126#discussion_r268856808 for a more detailed explanation of the technical requirement.

cc @ericstj

Reply:
In that link, @ericstj says:

> No we don't want to make that change in the ref. My point was that we shouldn't change the behavior here. It's actually less bad to change only in the implementation.
...
>  I'm ok with you modifying implementation only to Obsolete(..., false) only for the type

Did something change with that decision? I like @bartonjs's suggestion of

> To just acknowledge that not everything can be unified

is probably the best way forward here, for now.

Reply:
see https://github.com/dotnet/runtime/pull/65847#issuecomment-1051906802